### ISSUE TITLE
Course deprecation

### ISSUE THREAD

**Issue by jonathanbossenger** [WordPress/Learn#2868]
Some courses have become redundant or outdated with the initial learning pathways now published. To streamline our offerings and ensure we provide the most current content to our learners, @westnz and @jonathanbossenger want to deprecate the following courses and direct users to the relevant learning pathway.

- https://learn.wordpress.org/course/simple-site-design-with-full-site-editing/
- https://learn.wordpress.org/course/part-2-personalized-site-design-with-full-site-editing-and-theme-blocks/
- https://learn.wordpress.org/course/part-3-advanced-site-design-with-full-site-editing-site-editor-templates-and-template-parts/
- https://learn.wordpress.org/course/developing-with-the-wordpress-rest-api/
- https://learn.wordpress.org/course/converting-a-shortcode-to-a-block/
- https://learn.wordpress.org/course/a-developers-guide-to-block-themes-part-1/
- https://learn.wordpress.org/course/a-developers-guide-to-block-themes-part-2/

**Comment by jonathanbossenger** [comment]
@adamwoodnz, what is the required process to redirect content on Learn.WordPress.org? Are we using a redirection plugin, or is this handled as 301 redirects on the server (.htaccess or similar)?

**Comment by adamwoodnz** [comment]
> @adamwoodnz, what is the required process to redirect content on Learn.WordPress.org? Are we using a redirection plugin, or is this handled as 301 redirects on the server (.htaccess or similar)?

They are handled [here](https://github.com/WordPress/Learn/blob/trunk/wp-content/plugins/wporg-learn/inc/redirects.php#L47)

If you could detail which learning pathway the redirect should point to for each deprecation, someone can pick this up.

**Comment by jonathanbossenger** [comment]
Got it, thanks @adamwoodnz I can also create a PR if needed :grin: 

@westnz could you please add the learning pathways you want to redirect your courses to?

Thanks

**Comment by westnz** [comment]
Thank you, both.

I would like the courses to redirect to this URL: https://learn.wordpress.org/learning-pathway/user/

**Comment by jonathanbossenger** [comment]
Noting the redirections here, I'll do my best to create a PR later this week:

| Old URL                                                                                                                         | New URL                                                          |
|---------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------|
| https://learn.wordpress.org/course/simple-site-design-with-full-site-editing/                                                   | https://learn.wordpress.org/learning-pathway/user/               |
| https://learn.wordpress.org/course/part-2-personalized-site-design-with-full-site-editing-and-theme-blocks/                     | https://learn.wordpress.org/learning-pathway/user/               |
| https://learn.wordpress.org/course/part-3-advanced-site-design-with-full-site-editing-site-editor-templates-and-template-parts/ | https://learn.wordpress.org/learning-pathway/user/               |
| https://learn.wordpress.org/course/developing-with-the-wordpress-rest-api/                                                      | https://learn.wordpress.org/course/beginner-wordpress-developer/ |
| https://learn.wordpress.org/course/converting-a-shortcode-to-a-block/                                                           | https://learn.wordpress.org/course/beginner-wordpress-developer/ |
| https://learn.wordpress.org/course/a-developers-guide-to-block-themes-part-1/                                                   | https://learn.wordpress.org/course/intermediate-theme-developer/ |
| https://learn.wordpress.org/course/a-developers-guide-to-block-themes-part-2/                                                   | https://learn.wordpress.org/course/intermediate-theme-developer/ |

**Comment by jonathanbossenger** [comment]
PR has been created.

**Comment by adamwoodnz** [comment]
PR has shipped and I've updated all these Courses to be drafts. Feel free to trash if you like.

**Comment by westnz** [comment]
Many thanks, Adam and Jonathan!

**Comment by jonathanbossenger** [comment]
@adamwoodnz I'm guessing you set the courses to draft programmatically somehow, not via the Courses UI. It seems that doing this doesn't also set the lessons in the course to draft. Would you be able to do this for us as well?

**Comment by adamwoodnz** [comment]
> @adamwoodnz I'm guessing you set the courses to draft programmatically somehow, not via the Courses UI. It seems that doing this doesn't also set the lessons in the course to draft. Would you be able to do this for us as well?

No, I just used the Courses admin. That's frustrating! I see these lessons are still accessible on the frontend, because we are explicitly allowing lessons from unpublished courses to be viewed like standalone lessons. I guess this kind of deprecation won't be common in future, so shouldn't change that approach? For these I think we'll need to manually unpublish them unfortunately.

**Comment by westnz** [comment]
Thanks Adam. I went ahead and manually unpublished all the lessons in the three user courses. 😅

**Comment by jonathanbossenger** [comment]
> No, I just used the Courses admin. That's frustrating! 

Ah, indeed, I would have also assumed it unpublished the lessons if you did it that way. Apologies. I'll unpublish the lessons in my developer courses today.
